components/Spinner.vue

```diff
@@ -0,0 +1,8 @@
+<template>
```

Comment:
This component needs a multi-word name to follow the essential recommendation at https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential

Reply:
Thanks, I have updated the name of this component.